Lib/fontmake/__main__.py

```diff
@@ -335,14 +335,22 @@ def main(args=None):
         help="Enable ufoLib validation on reading/writing UFO files. It is "
         "disabled by default",
     )
-    outputGroup.add_argument(
+    compatibilityGroup = outputGroup.add_mutually_exclusive_group()
+    compatibilityGroup.add_argument(
         "--check-compatibility",
         action="store_true",
         help="Check if the source files are interpolatable. It is "
         "disabled by default, but enabled when building variable fonts "
         "or what the 'Enforce Compatibility Check' custom parameter is "
         "set on a Glyphs file",
     )
+    compatibilityGroup.add_argument(
+        "--no-check-compatibility",
+        action="store_true",
+        dest="disable_compatibility_check",
```

Comment:
how about you make the existing `check_compatibility` a 3-state option instead of adding a new `disable_compatibility_check` bool parameter: i.e. `None` by default, which means `True` for interpolable outputs, `False` otherwise; and then explicit `True` / `False`, respectively controlled by the two CLI flags --check-compatiblity / --no-check-compatibility; if omitted, default to None.